docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -164,9 +206,9 @@ Teleport > Waiting for required participants...
 ```
 
 Jeff's session is paused, waiting for the required observers.
-
-Now Alice with the `auditor` role joins as a moderator and
-the session can begin.
+When Alice—a user assigned the `auditor` role—joins the waiting session 
```

Comment:
```suggestion
When Alice, a user assigned the `auditor` role, joins the waiting session 
```

Or, alternatively:

```suggestion 
_When Alice (a user assigned the `auditor` role) joins the waiting session_.
```

Not sure if this is grammatically more correct, but it sure helps me to read the sentence. With the hyphenated version I see words like "role-joins" and ask myself "what in the world does it mean to role-join?"